erts/emulator/test/process_SUITE.erl

```diff
@@ -2332,6 +2351,821 @@ handle_event(Event, Pid) ->
     Pid ! Event,
     {ok, Pid}.
 
+
+spawn_request_bif(Config) when is_list(Config) ->
+    io:format("Testing spawn_request() on local node...~n"),
+    spawn_request_bif_test(true, node()),
+    io:format("Testing spawn_request() on local node with Node...~n"),
+    spawn_request_bif_test(false, node()),
+    {ok, Node} = start_node(Config),
+    io:format("Testing spawn_request() on remote node...~n"),
+    spawn_request_bif_test(false, Node),
+    stop_node(Node),
+    ok.
+                       
+spawn_request_bif_test(Local, Node) ->
```

Comment:
Should we add a testcase with a "large" enough argument to trigger dist message fragmentation?